pkg/runner/runner.go

```diff
@@ -160,10 +160,10 @@ func (r *Runner) Run(ctx context.Context) error {
 		}
 
 		logger.Infof("Running as detached process (PID: %d)", os.Getpid())
+	} else {
+		logger.Info("Press Ctrl+C to stop or wait for container to exit")
```

Comment:
There's not much point in printing this in a detached process, so I moved it inside an else block